installer_scripts/install-yb-voyager

```diff
@@ -180,7 +180,7 @@ centos_main() {
 	output "Installing RPM dependencies."
 	$YUM which wget git gcc make 1>&2
 	$YUM https://download.postgresql.org/pub/repos/yum/reporpms/EL-${majorVersion}-x86_64/pgdg-redhat-repo-latest.noarch.rpm 1>&2 || true
-	$YUM postgresql14 1>&2
+	$YUM postgresql16 1>&2
```

Comment:
Should we also be modifying voyager code to look for pg_dump>=16? cc @priyanshi-yb